### PR TITLE
Remodeled some modules

### PR DESCRIPTION
**Pull request by rajaram5**
I remodeled biobanks and consent modules. I removed optional nodes from disability module. For this release it is better to keep the model simpler

### CODE REVIEW

File: example-data/turtle/sio-model/consent.ttl [hunk 1]
```diff
@@ -2,17 +2,24 @@
...
+    sio:SIO_000356 :process_2 .
 
-:process_ a obo:ICO_0000196 ;
-    sio:SIO_000229 <http://example.org/my_consent_document.pdf> ;
-    prov:startedAtTime "2020-10-18T13:00:00"^^xsd:dateTime .
+:process_2 a obo:ICO_0000181 ;
+    sio:SIO_000229 :output_2 ;
+    sio:SIO_000680 "2020-10-18T13:03:00"^^xsd:dateTime .
+    
+:output_1 a sio:SIO_000785, obo:NCIT_C49488 .
```
Comment (NuriaQueralt) [inline example-data/turtle/sio-model/consent.ttl:23]: The turtle is perfect. Regarding the model, I agree with it. One minor concern is the definition of *output* using ncit:yes that is a subclass of 'activity' (kind of a process), while as output one would expect information/data entities. For the next release, as alternative to using NCIT could be: 1) 'yes answer to question' (obo:OPMI_0000023), which is an information entity in the Ontology of Precision Medicine and Investigation; and optionally refer to 'full agreement' (sio:001179), which is an informational quality:

'yes answer to question' (http://purl.obolibrary.org/obo/OPMI_0000023)
 subClassOf
'information content entity'
that 'has value' value "yes"
and 'refers to' some 'agreement quality'

'full agreement'
subClassOf
'agreement quality'

'person'
subClassOf
'entity'
and 'has quality' some 'agreement quality'


In RDF:
:person_ a sio:person;
  sio:has-quality sio:full-agreement .

:output_ a opmi:yes-answer-to-question;
   sio:refers-to sio:full-agreement .

File: example-data/turtle/sio-model/consent.ttl [hunk 2]
```diff
@@ -2,17 +2,24 @@
...
 
-<http://example.org/my_consent_document.pdf> a sio:SIO_000340, obo:OBIB_0000488 .
+:output_2 a sio:SIO_000785, obo:NCIT_C49487.
```
Comment (NuriaQueralt) [inline example-data/turtle/sio-model/consent.ttl:25]: Same as before. For the next release, as alternative to using NCIT could be: 1) 'no answer to question' (obo:OPMI_0000024), which is an information entity in the Ontology of Precision Medicine and Investigation; and optionally refer to 'full disagreement' (sio:001181), which is an informational quality:

'no answer to question' (http://purl.obolibrary.org/obo/OPMI_0000024)
 subClassOf
'information content entity'
that 'has value' value "no"
and 'refers to' some 'agreement quality'

'full disagreement'
subClassOf
'agreement quality'

'person'
subClassOf
'entity'
and 'has quality' some 'agreement quality'


In RDF:
:person_ a sio:person;
  sio:has-quality sio:full-disagreement .

:output_ a opmi:no-answer-to-question;
   sio:refers-to sio:full-disagreement .

File: shex/sio-model/8_Consent.shex [hunk 3]
```diff
@@ -1,26 +1,25 @@
...
 
-:consentShape IRI {
-  a [ obo:OBIB_0000488 ] ;
-  a [ sio:SIO_000340 ]
-}
\ No newline at end of file
+:outputShape IRI {
+  a [sio:SIO_000785];
+  a [obo:NCIT_C49488 obo:NCIT_C49487]
+}
```
Comment (NuriaQueralt) [inline shex/sio-model/8_Consent.shex]: The shex is perfect for the rdf file!

**Review by NuriaQueralt — APPROVED** [review]

File: example-data/turtle/sio-model/biobanks.ttl [hunk 4]
```diff
@@ -2,16 +2,25 @@
...
+ :role_1 a obo:OBI_0000093 ;
+    sio:SIO_000356 :sampling_process_ .
 
 :specimen_ a sio:SIO_010412 ;
-  sio:SIO_000228 :role_ .
+  sio:SIO_000228 :role_2 .
 
-:role_ a obo:OBI_0000740 ;
-  sio:SIO_000356 :storing_process_ .
+:role_2 a obo:OBI_0000112 ;
```
Comment (NuriaQueralt) [inline example-data/turtle/sio-model/biobanks.ttl:16]: specimen_role! I missed it! GREAT that you found it!!!

File: example-data/turtle/sio-model/biobanks.ttl [hunk 5]
```diff
@@ -2,16 +2,25 @@
...
+:biobank_ a obo:OMIABIS_0000010 ;
+  dc:title "UK BioBank"^^xsd:string ;
+  sio:SIO_000228 :role_3 .
+  
+:role_3 a obo:OBI_0000947 ;
+  sio:SIO_000356 :sampling_process_ .
 
-:biobank_ a obo:OMIABIS_0001078 ;
-  sio:SIO_000300 "UK BioBank"^^xsd:string .
+:sampling_process_ a obo:OMIABIS_0000061 .
```
Comment (NuriaQueralt) [inline example-data/turtle/sio-model/biobanks.ttl:26]: Looks very good now!

**Review by NuriaQueralt — APPROVED** [review]

**Review by sxzhang1201 — APPROVED** [review]
All ShEx conforms to figures and example data are syntactically correct.

File: example-data/turtle/sio-model/consent.ttl (hunk 1 above)
Comment (rajaram5) [reply example-data/turtle/sio-model/consent.ttl:23]: Nice catch @NuriaQueralt I overlooked the super class. I will replace NCIT with proposed classes

**Review by NuriaQueralt — APPROVED** [review]

**Review by NuriaQueralt — APPROVED** [review]

**Review by NuriaQueralt — APPROVED** [review]

**Review by NuriaQueralt — APPROVED** [review]

**Comment by pabloalarconm** [comment]
All comments @NuriaQueralt mentioned about new changes are fine with me

**Comment by rajaram5** [comment]
Thanks all for the review. I fixed that PR issues I will merge the changes

**Pull request merged**